CHANGELOG.md

```diff
@@ -2,7 +2,27 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
-## [v8.2.1](https://github.com/puppetlabs/puppetlabs-apache/tree/v8.2.1) (2022-09-26)
+## [v8.3.0](https://github.com/puppetlabs/puppetlabs-apache/tree/v8.3.0) (2022-10-28)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-apache/compare/v8.2.1...v8.3.0)
+
+### Added
+
+- Update EL8+ and Debian SSL defaults [\#2336](https://github.com/puppetlabs/puppetlabs-apache/pull/2336) ([ekohl](https://github.com/ekohl))
+- Support setting SSLProxyCipherSuite on mod\_ssl [\#2335](https://github.com/puppetlabs/puppetlabs-apache/pull/2335) ([ekohl](https://github.com/ekohl))
+
+### Fixed
+
+- Make serveradmin an optional parameter and use it [\#2338](https://github.com/puppetlabs/puppetlabs-apache/pull/2338) ([ekohl](https://github.com/ekohl))
+- pdksync - \(CONT-189\) Remove support for RedHat6 / OracleLinux6 / Scientific6 [\#2326](https://github.com/puppetlabs/puppetlabs-apache/pull/2326) ([david22swan](https://github.com/david22swan))
+- pdksync - \(CONT-130\) Dropping Support for Debian 9 [\#2322](https://github.com/puppetlabs/puppetlabs-apache/pull/2322) ([jordanbreen28](https://github.com/jordanbreen28))
+
+### UNCATEGORIZED PRS; LABEL THEM ON GITHUB
```

Comment:
I labeled and now running: https://github.com/puppetlabs/puppetlabs-apache/actions/runs/3345071806

Reply:
I wonder why the action is taking more than an hour now while the previous run just took 7 minutes.

Reply:
It varies widely

Reply:
I cancelled it after 2 nearly hours and kicked off https://github.com/puppetlabs/puppetlabs-apache/actions/runs/3345774314.

I did notice that the job itself runs `bundle install` from inside a container image, which is weird. Normally a container already has everything.